frontend/website/src/Testnet.re

```diff
@@ -365,14 +322,9 @@ module Styles = {
       fontSize(`rem(0.625)),
       media(
         Style.MediaQuery.notMobile,
-        [
-          height(`rem(25.)),
-          paddingTop(`rem(4.6875)),
-          paddingBottom(`rem(4.6875)),
-          paddingLeft(`rem(2.1875)),
-          paddingRight(`rem(2.1875)),
-          fontSize(`rem(1.)),
-        ],
+        Style.paddingX(`rem(2.1875))
+        @ Style.paddingY(`rem(4.6875))
```

Comment:
you can also use `padding2(~v=..., ~h=...)`, meaning you don't need all those @s.